transformers/events/log_min_sell/entity.go

```diff
@@ -0,0 +1,16 @@
+package log_min_sell
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type LogMinSellEntity struct {
+	Id              *big.Int
```

Comment:
I think we can probably delete this field from the entity

Reply:
Those aren't required? Kindda surprised.

Reply:
Yeah the entity is just used for unpacking blockchain data into Go, so in theory it should only have the fields that are defined on the event (pay_gem, min_amount). By convention, we've been adding important metadata that needs to be inserted into postgres (header_id, log_id, contract_address), but don't do that for the database ID since those are created automatically (and occasionally the event will have its own "Id" that the entity needs to declare in order to unpack, and we end up inserting those as offer_id/bid_id/etc).